### ISSUE TITLE
permissions issues

### ISSUE THREAD

**Issue by aderbenev** [ChannelFinder/recsync#9]
When using RecCeiver which was installed as a Debian (7) package, launching `sudo /etc/init.d/recceiver start` results in `.pid` file being created in `/var/run/`. Then, `sudo /etc/init.d/recceiver stop` causes `Warning: No permission to delete pid file` warning. Modifying the init.d script to create the `.pid` file in a directory writable by `recceiver` fixes the issue and the file is deleted properly when the service is stopped.

A similar problem appears when trying to make RecCeiver use an sqlite database file. Even if the database file is owned by `recceiver`, an `sqlite.OperationalError: unable to open database file` is thrown by the code once a client connects. Placing the database file into a folder writable by `recceiver` fixes the issue, presumably because ["SQLite needs to be able to create additional files in it in order to handle things like the commit log"](https://stackoverflow.com/questions/4636970/sqlite3-operationalerror-unable-to-open-database-file).

**Comment by mdavidsaver** [comment]
The pid file permissions is definitely a bug.  As for the sqlite db file, what path is it trying to use?

**Comment by aderbenev** [comment]
As the default configuration does not have sqlite plugin enabled, it tried to use whatever path I gave it in `.conf` file. In my case, it was a database file I created in my home folder using the provided schema. While the file was owned by me, the code was throwing "attempt to write a readonly database" exception. After I assigned file permissions to `recceiver`, the exception switched to the `unable to open database file` message. Once I created a separate directory owned by `recceiver` and put the database file in, the error went away.

As a side note, sqlite instructions in [README](https://github.com/mdavidsaver/recsync/blob/master/debian/README) are misplaced. The init command has to be:

```
$ sqlite3 -init usr/share/doc/recceiver/recceiver.sqlite3 /path/to/somefile.db .exit
```

P.S. I can add that the issue with sqlite access only appears when `twistd` is used with `--uid=recceiver` as otherwise it runs as root without problems.

**Comment by mdavidsaver** [comment]
> P.S. I can add that the issue with sqlite access only appears when twistd is used with --uid=recceiver as otherwise it runs as root without problems.

Some would consider daemons running as root to be a problem in and of itself ;)

**Comment by aderbenev** [comment]
Not that I am advocating for solving everything with elevating processes to root, just providing clues that made me think the issue was a permission problem T_T

**Comment by mdavidsaver** [comment]
The PID file permissions issue should be resolved with 146627d4f04fba5ba42aa96d59ebe8829b8e482a.  I've updated the README related to the sqlite setup as well.  Let me know if this improves things.

**Comment by aderbenev** [comment]
I have tested the latest version and can confirm that the PID file is now correctly removed when the service is stopped. New README also now looks sufficient to make database work.
